rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/config/ConnectConfig.java

```diff
@@ -113,7 +113,7 @@ public void setBrokerSuspendMaxTimeMillis(int brokerSuspendMaxTimeMillis) {
     /**
      * Http port for REST API.
      */
-    private int httpPort = 8081;
+    private int httpPort = 8082;
```

Comment:
why change to 8082?

Reply:
I notice the port above README and config file is 8082

Reply:
I notice the port above README and config file is 8082